plugins/reporters/cyclonedx/src/main/kotlin/CycloneDxReporter.kt

```diff
@@ -148,10 +147,18 @@ class CycloneDxReporter : Reporter {
         val dataLicense = config.options[OPTION_DATA_LICENSE] ?: DEFAULT_DATA_LICENSE.id
         val createSingleBom = !config.options[OPTION_SINGLE_BOM].isFalse()
 
-        val outputFileFormats = config.options[OPTION_OUTPUT_FILE_FORMATS]
+        val outputFileExtensions = config.options[OPTION_OUTPUT_FILE_FORMATS]
             ?.split(",")
-            ?.mapTo(mutableSetOf()) { FileFormat.valueOf(it.uppercase()) }
-            ?: setOf(FileFormat.XML)
+            ?.mapTo(mutableSetOf()) { it.trim().lowercase() }
+            .orEmpty()
```

Comment:
Instead of `orEmpty()` we can move the `ifEmpty` here. This removes the need for the  `validFileExtensions` variable entirely (remove lines 160 + 161) and to operate on ` outputFileExtensions` only.

Reply:
It was a tiny bit more trick as also the null case needs to be handled, but done.

Reply:
Thanks!